File: Python/card.py
from payment import Payment

class Card(Payment):
    card_number = str
    card_date = str
    cvv = int

    def __init__(self, id, card_number, card_date, cvv):
        super(Card, self).__init__(id)
        self.card_number = card_number
        self.card_date = card_date
        self.cvv = cvv

File: JS/Card.js
class Card extends Payment {
    constructor(id, cardNumber, cardDate, cvv) {
        //? "super" makes reference to atributes and methods of the super class or father class
        super(id);
        //? While "this" makes reference to the class you are right know or the child class
        this.cardNumber = cardNumber;
        this.cardDate = cardDate;
        this.cvv = cvv;
    }
}
